### ISSUE TITLE
Posterior trace - nan in output

### ISSUE THREAD

**Issue by grafau** [dipetkov/eems#21]
Hi there!

While I was using eems and plotting functions, i managed to get migration maps but no diagnostic figures are being generated. I got the following error:

Error in plot.window(...) : need finite 'ylim' values
In addition: Warning messages:
1: In min(x) : no non-missing arguments to min; returning Inf
2: In max(x) : no non-missing arguments to max; returning -Inf

I figured posteriors might be in mcmcpilogl.txt file and mine has only 'nan' in second column. What could cause such behaviour? Any help would be appreciated. In general, I couldn't find the annotation for output files in the manual. It would be super handy!

Thanks!
Rafal

**Comment by dipetkov** [comment]
You are correct that `mcmcpilogl.txt` stores the prior in the first column and the likelihood in the second column (on the log scale, after burn-in and thinning; the log posterior is column 1 + column 2). 

So if the likelihood is `nan` something has definitely gone wrong. If you've saved the output or if you can run eems again, can you track the likelihood from its initial value? Is it `nan` at the start or it becomes `nan` after some number of iterations. (You can set `numBurnIter = 0` and `numThinIter = 0`, so that information about every iteration is printed while runeems is running.)

Here is some example output:

```
......

Initial log prior: 418.93
Initial log llike: 158288.63

......

Ending iteration 1 ......
         Log prior = 367.01
         Log llike = 163507.44

......
